### DISCUSSION TITLE
Mkdocs Material for documentation?

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#634] [Ideas]
Just a FYI - check out https://github.com/squidfunk/mkdocs-material for a pretty impressive open source framework for documentation. The included search capabilities could really help Netlab users find their way

**Comment by ipspace** [comment]
OOH, I like it. OTOH, mkdocs seems to be lacking basic stuff like "page ToC within a page" that we use in almost all large documents. Then there's cross-document named anchors, automatic insertion of page/section titles based on link URL etc.

You did get me the idea to publish the documentation on GitHub pages though ;) Thank you!

**Reply by jbemmel** [reply]
You mean like this? https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#without_5
![image](https://user-images.githubusercontent.com/2031627/201340644-3c5f08ef-8c50-4df8-8450-247506bcf18a.png)

**Reply by ipspace** [reply]
That could work ;) Thank you!

**Comment by ipspace** [comment]
Just figured out that Sphinx also includes some sort of search functionality. Open https://netlab.tools/ and try the search box in the top left corner of the screen.
